### DISCUSSION TITLE
Permission

### DISCUSSION THREAD

**Discussion by fatabek** [phpcollab/phpcollab#80] [Ideas]
Can I change or define a new permisson level ? For example ;
User can se only his/her project and task but do not change anything ? It is possible ?

Tahanks a lot

**Comment by mindblender** [comment]
Hi @fatabek I did not see your question, and I apologize!

There is a "beta" piece that only allows you to see your projects items. It is in the `includes/settings.php` file (https://github.com/phpcollab/phpcollab/blob/master/includes/settings_default.php#L178)  Not sure if that would give you what you are looking for or not.
